### ISSUE TITLE
r35 - Linux - MON no longer scrolls

### ISSUE THREAD

**Issue by Usoilworker** [commanderx16/x16-emulator#225]
When using MON I am no longer able to scroll thru memory with the cursor up/down.
Using the F3 F5 keys as described in the instructions executes Load/save.

**Comment by mist64** [comment]
Yes, I had to disable this for now. The monitor had to be moved to its own bank, and integrating with KERNAL is now harder.
